vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -173,29 +173,57 @@ struct RootShadowNode final : public ShadowNodeBase {
   }
 };
 
-void NativeUIManager::setHost(facebook::react::INativeUIManagerHost *host) {
+void NativeUIManager::setHost(INativeUIManagerHost *host) {
   m_host = host;
 }
 
-facebook::react::ShadowNode *NativeUIManager::createRootShadowNode(facebook::react::IReactRootView *pReactRootView) {
+ShadowNode *NativeUIManager::createRootShadowNode(facebook::react::IReactRootView *pReactRootView) {
   return new RootShadowNode(pReactRootView, m_host);
 }
 
-void NativeUIManager::destroyRootShadowNode(facebook::react::ShadowNode *node) {
+void NativeUIManager::destroyRootShadowNode(ShadowNode *node) {
   delete node;
 }
 
-void NativeUIManager::AddRootView(
-    facebook::react::ShadowNode &shadowNode,
-    facebook::react::IReactRootView *pReactRootView) {
-  auto xamlRootView = static_cast<IXamlRootView *>(pReactRootView);
+int64_t NativeUIManager::AddMeasuredRootView(facebook::react::IReactRootView *rootView) {
+  auto tag = m_nextRootTag;
+  m_nextRootTag += RootViewTagIncrement;
+
+  int64_t width = rootView->GetActualWidth();
+  int64_t height = rootView->GetActualHeight();
+
+  m_host->RegisterRootView(rootView, tag, width, height);
+
+  // TODO: call UpdateRootNodeSize when ReactRootView size changes
```

Comment:
is this happening in this PR or in a followup?

Reply:
This was an existing comment that just got moved with the file.